docs/pages/guides/tutorials/Hello-Kubernetes-with-Dapr.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: default
+title: Hello Kubernetes with Dapr
+abstract: |
+  Dive into this step-by-step tutorial on deploying Dapr in Kubernetes using Meshery Playground. Learn to deploy a Python app for generating messages and a Node app for consuming and persisting them, all orchestrated with Dapr for enhanced application integration.
+permalink: guides/tutorials/hello-kubernetes-with-dapr
+redirect_from: guides/tutorials/Hello-Kubernetes-with-Dapr
```

Comment:
Don't include capital letters in URLs.

Lowercase this reference and add a forward slash to the end.